docs/developer-guides/local-development.md

```diff
@@ -0,0 +1,20 @@
+# Local Development Setup
+
+To contribute to Cluster Registry, you need to set up a local environment.
+
+## Initial setup
+
+Cluster Registry development stack consist of the following applications that runs in Docker containers:
+
+* cluster-registry-api
+* cluster-registry-client
+* aws sqs
+* aws dynamoDb
+* a local k8s cluster created with kind
```

Comment:
Need to add `local oidc mocking provider`. Was added recently in https://github.com/adobe/cluster-registry/pull/3.
Maybe also it is worth to specity `local aws sqs` + `local aws dynamoDb`